lms/envs/devstack_with_worker.py

```diff
@@ -19,6 +21,29 @@
 # Require a separate celery worker
 CELERY_ALWAYS_EAGER = False
 
+# Periodic tasks can be created automatically at project start (when celery process is started).
+# To not create them manually in admin (but they still will be shown in admin),
+# we can define them in settings.py in such way.
+CELERYBEAT_SCHEDULE = {
+    'student_counter': {
+    'task': 'openedx.core.djangoapps.global_statistics.tasks.count_data',
+    'schedule': 20,
+    }
+}
+
+# URL to send data within periodic task processing.
+PERIODIC_TASK_POST_URL = 'http://192.168.1.139:7000/receive/'
```

Comment:
I moved it to `lms.envs.json` file, I think at least currently, it's more convenient to use it in such way for local development.